kalite/distributed/settings.py

```diff
@@ -201,3 +201,34 @@ def USER_FACING_PORT():
 ########################
 
 LOCKDOWN = getattr(local_settings, "LOCKDOWN", False)
+
+
+########################
+# Screenshots
+########################
+
+from django.conf import settings
+PROJECT_PATH = os.path.realpath(getattr(local_settings, "PROJECT_PATH", settings.PROJECT_PATH)) + "/"
+
+SCREENSHOTS_OUTPUT_PATH = os.path.join(os.path.realpath(PROJECT_PATH), "..", "data", "screenshots")
+SCREENSHOTS_EXTENSION = ".png"
+
+SCREENSHOTS_DATABASE_NAME = "screenshot-data.sqlite"
+SCREENSHOTS_DATABASE_PATH = os.path.join(SCREENSHOTS_OUTPUT_PATH, SCREENSHOTS_DATABASE_NAME)
+
+SCREENSHOTS_JSON_PATH = os.path.join(os.path.dirname(__file__), "data")
+SCREENSHOTS_JSON_FILE = os.path.join(SCREENSHOTS_JSON_PATH, 'screenshots.json')
+SCREENSHOTS_ROUTER = 'default'
+SQLITE3_ENGINE = 'django.db.backends.sqlite3'
+
+if 'screenshots' in sys.argv:
+    # use another sqlite3 database for the screenshots
+    DATABASES = {
+        SCREENSHOTS_ROUTER: {
+            "ENGINE": getattr(local_settings, "DATABASE_TYPE", SQLITE3_ENGINE),
+            "NAME": SCREENSHOTS_DATABASE_PATH,
```

Comment:
@aronasorman so you're saying if this setting has value ":memory:" it'll run in memory? (I can't find documentation of this... :( )